### PR TITLE
Added a new implementation of naming conventions

### PR DESCRIPTION
**Pull request by BenjaminRCooper**
As stated in #13 and #8, you guys are looking for a new naming convention to fit inline with how you currently build your width utilities and allow for extension of adding the responsive suffixes.

Thought I would put forward this in the hope it sparks some new conversation or it becomes useful at some point

### CODE REVIEW

File: _trumps.spacing.scss [hunk 1]
```diff
@@ -177,13 +177,13 @@ $inuit-enable-paddings--none:           false !default;
...
      */
 
-    .#{$inuit-spacing-namespace}u--m--  { margin:           -$inuit-margin--tiny !important; }
-    .#{$inuit-spacing-namespace}u--mt-- { margin-top:       -$inuit-margin--tiny !important; }
-    .#{$inuit-spacing-namespace}u--mr-- { margin-right:     -$inuit-margin--tiny !important; }
-    .#{$inuit-spacing-namespace}u--mb-- { margin-bottom:    -$inuit-margin--tiny !important; }
-    .#{$inuit-spacing-namespace}u--ml-- { margin-left:      -$inuit-margin--tiny !important; }
-    .#{$inuit-spacing-namespace}u--mh-- { margin-right:     -$inuit-margin--tiny !important; margin-left:     -$inuit-margin--tiny !important; }
-    .#{$inuit-spacing-namespace}u--mv-- { margin-top:       -$inuit-margin--tiny !important; margin-bottom:   -$inuit-margin--tiny !important; }
+    .#{$inuit-spacing-namespace}u--m--1\/4  { margin:           -$inuit-margin--tiny !important; }
```
Comment (jmiguel-localvox) [inline _trumps.spacing.scss]: You've got an extra `--` in these classnames (all the way to `4x`), `1x` doesn't have this.

File: _trumps.spacing.scss [hunk 2]
```diff
@@ -177,13 +177,13 @@ $inuit-enable-paddings--none:           false !default;
...
+    .#{$inuit-spacing-namespace}u--mt--1\/4 { margin-top:       -$inuit-margin--tiny !important; }
```
Comment (jmiguel-localvox) [inline _trumps.spacing.scss]: Why not have an `x` next to the fractional units as you do for the whole units?

Comment (BenjaminRCooper) [inline _trumps.spacing.scss]: Don't see the need to be honest.

`x` signifies that it is a multiple base spacing unit which is `$inuit-margin`.

File: _trumps.spacing.scss (hunk 1 above)
Comment (BenjaminRCooper) [inline _trumps.spacing.scss]: Thanks, updated the PR.

File: _trumps.spacing.scss (hunk 2 above)
Comment (jmiguel-localvox) [inline _trumps.spacing.scss]: The only places where the base margin is used is the `1x` variant. All others use some kind of `$inuit-margin--variant`. I would argue that it's not necessary to have the `1x` naming at all, but where the other variants are used to use an `x` in the name. Even if we are halving or quartering the value, we're still making the margin in these cases proportional to some other value.

Comment (BenjaminRCooper) [inline _trumps.spacing.scss]: But all the variants derive from the same base unit so it still applies. Both solutions work as fractions are still proportional of the the base unit, but I can understand your argument. I would just say it adds unnecessary characters as it is understandable as is.

**Comment by csshugs** [comment]
Since this repository is now officially deprecated, I close this.
If you still consider this as relevant, please open a new issue in the [new repository](https://github.com/inuitcss/inuitcss).

Thank you very much for your time and effort of providing this PR, though!

**Pull request closed without merging**